plugin-packs/postcss-preset-env/src/index.js

```diff
@@ -99,18 +101,27 @@ const plugin = opts => {
 	// browsers supported by the configuration
 	const supportedBrowsers = browserslist(browsers, { ignoreUnknownVersions: true });
 
-	// features supported by the stage and browsers
-	const supportedFeatures = stagedFeatures.filter(
-		feature => feature.id in features
-			? features[feature.id]
-			: supportedBrowsers.some(
-				supportedBrowser => browserslist(feature.browsers, {
-					ignoreUnknownVersions: true,
-				}).some(
-					polyfillBrowser => polyfillBrowser === supportedBrowser,
-				),
-			),
-	);
+	// - features supported by the stage
+	// - features with `true` or with options
+	// - required for the browsers
+	// - having "importFrom" or "exportTo" options
+	const supportedFeatures = stagedFeatures.filter((feature) => {
+		if (feature.id in features) {
+			return features[feature.id];
```

Comment:
I am pretty sure that this code is non ideal, but this bit (109-111) is equivalent to how it was before.

In practice it means that you lose the dynamic nature of postcss-preset-env if you set any explicity plugin options.

We should start a list of changes for a next major version to cleanup API that is hard to use and hard to implement.
Overloading enabling/disabling a plugin on top of providing plugin options is just not the best API design. :)